src/client/Microsoft.Identity.Client/MsalException.cs

```diff
@@ -22,6 +22,12 @@ public class MsalException : Exception
     {
         private string _errorCode;
 
+        /// <summary>
+        /// Returns if the the flow is retryable for the exception.
+        /// </summary>
+        /// <returns>A bool value to indicate if retry can be performed in case of the exception.</returns>
+        public bool IsRetryable { internal set; get; }
```

Comment:
order should be {get; internal set;}

Reply:
In fact, it should be just `IsRetryable { get; }`, because you can always set an auto-property from the ctor, i.e. `IsRetryable { get; }` is equivalent to `IsRetryable { get; private readonly set; }` (this syntax does not exist, but just to prove the point).